gemfiles/rails_5_1.gemfile

```diff
@@ -1,41 +1,40 @@
-# frozen_string_literal: true
-
 # This file was generated by Appraisal
 
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-gem 'rails', '~> 5.1.6'
+gem "rails", "~> 5.1.6"
 
 group :development, :test do
-  gem 'attr_encrypted'
-  gem 'figaro', git: 'https://github.com/laserlemon/figaro'
-  gem 'omniauth-facebook', git: 'https://github.com/mkdynamic/omniauth-facebook'
-  gem 'omniauth-github', git: 'https://github.com/intridea/omniauth-github'
-  gem 'omniauth-google-oauth2', git: 'https://github.com/zquestz/omniauth-google-oauth2'
-  gem 'rack-cors', require: 'rack/cors'
-  gem 'thor'
-  gem 'faker'
-  gem 'fuzz_ball'
-  gem 'guard'
-  gem 'guard-minitest'
-  gem 'minitest'
-  gem 'minitest-focus'
-  gem 'minitest-rails'
-  gem 'minitest-reporters'
-  gem 'mocha', '>= 1.5'
-  gem 'pry'
-  gem 'pry-remote'
-
-  gem 'rubocop', require: false
+  gem "attr_encrypted"
```

Comment:
why going back with double quotes? we fixed this with rubocop

Reply:
`bundle exec appraisal install` generates gem sets. I think the command is the reason. I could replace double quotes with editor. What do you think?

Reply:
well if it will happen all the time due to be autogenerated, leave as it is. I will continue reviewing this